packages/horizontal-slider-vdom/src/types/Renderer.ts

```diff
@@ -0,0 +1,36 @@
+export type Pragma = (
```

Comment:
Next component that we create we might want have a shared/common package for these types.

Reply:
Yep, I can already see multiple use cases for a shared package!